pkg/virt-controller/watch/vmi_test.go

```diff
@@ -356,13 +356,13 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			})
 		})
 
-		Context("when pod owned by virt-handler", func() {
-			It("should remove scheduling pod condition from the VirtualMachineInstance", func() {
+		Context("when Pod recovers from scheduling issues", func() {
```

Comment:
I do not sure about this tests, I think "shouldExpectPodHandover" not really must be part of the check, I expect something like:
- if the pod has scheduling false condition and the VM not, the condition must be added to the VM after the Execute
- if the pod does not have scheduling false condition and the VM has, the condition must be removed from the VM after the Execute

Reply:
Tried to address it with `ignorePodUpdates` and `testutils.IgnoreEvents`.